tests/test_restore_names.py

```diff
@@ -1,3 +1,36 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_should_return_none():
+    assert restore_names([]) is None
+
+
+def test_first_name_is_none_or_first_name_is_lost():
```

Comment:
Please, add separate tests for both cases (None and missing key).
In that case, it'd be easier to find where a bug has occurred if the function does something wrong